.evergreen/prepare-oidc-get-tokens-docker.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```

Comment:
Confirm that https://github.com/mongodb/specifications/pull/1569 has been merged into spec repo, before merging this PR.

Reply:
merged